packages/stark-ui/src/modules/pretty-print/components/pretty-print.component.spec.ts

```diff
@@ -342,100 +374,124 @@ describe("PrettyPrintComponent", () => {
+				const invalidRawTypescriptData: string = rawTypescriptData + "}";
+
+				hostComponent.data = invalidRawTypescriptData;
+				hostComponent.format = "typescript";
+				hostFixture.detectChanges();
+
+				expect(component.prettyString).toBe(invalidRawTypescriptData);
+
+				const preElement: HTMLPreElement | null = <HTMLPreElement>hostFixture.nativeElement.querySelector("pre");
+				expect(preElement).not.toBeNull();
+				expect(preElement.innerHTML).toContain("if (!data){data = 'could not calculate data';}return data;");
 			});
 		});
 
 		describe("Undefined format", () => {
 			beforeEach(() => {
-				hostComponent.data = rawTypeScriptData;
+				hostComponent.data = rawTypescriptData;
 				hostComponent.format = <any>undefined;
 				hostFixture.detectChanges();
 			});
 
 			it(shouldHaveInputs, () => {
-				expect(component.data).toBe(rawTypeScriptData);
+				expect(component.data).toBe(rawTypescriptData);
 				expect(component.format).toBeUndefined();
 				expect(component.enableHighlighting).toBeUndefined();
-			});
+			}); // the same test is performed with an unknown format
 
-			/* tslint:disable: no-identical-functions */ // the same test is performed an unkowm format
-			it("should leave the raw data unformatted when the format is not defined", () => {
-				expect(component.prettyString).toBe(rawTypeScriptData);
+			/* tslint:disable: no-identical-functions */ it("should leave the raw data unformatted when the format is not defined", () => {
+				expect(component.prettyString).toBe(rawTypescriptData);
 
 				const preElement: HTMLPreElement | null = <HTMLPreElement>hostFixture.nativeElement.querySelector("pre");
 				expect(preElement).not.toBeNull();
-				/* tslint:disable: no-duplicate-string */
 				expect(preElement.innerHTML).toContain("function calculateData(seed:any, operationFn:Function):any");
-				/* tslint:enable: no-duplicate-string */
 			});
 			/* tslint:enable: no-identical-functions */
 		});
 
 		describe("Unknown Format string", () => {
 			beforeEach(() => {
-				hostComponent.data = rawTypeScriptData;
+				hostComponent.data = rawTypescriptData;
 				hostComponent.format = "UnknownFormat";
 				hostFixture.detectChanges();
 			});
 
 			it(shouldHaveInputs, () => {
-				expect(component.data).toBe(rawTypeScriptData);
+				expect(component.data).toBe(rawTypescriptData);
 				expect(component.format).toBe("unknownformat");
 				expect(component.enableHighlighting).toBeUndefined();
-			});
+			}); // the same test is performed an undefined format
 
-			/* tslint:disable: no-identical-functions */ // the same test is performed an undefined format
-			it("should leave the raw data unformatted when the format is an unrecognised string", () => {
-				expect(component.prettyString).toBe(rawTypeScriptData);
+			/* tslint:disable: no-identical-functions */ it("should leave the raw data unformatted when the format is an unrecognised string", () => {
```

Comment:
The format of these 2 comment lines is weird and of course is due to Prettier. I suggest to change them like this:

```
// the same test is performed with an unknown format
// tslint:disable-next-line: no-identical-functions
it (......)
```